cluster/codon/execute/test/test/scripts/step1.test.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
Argh... we need to remove this. It's my fault, I accidentaly pushed at some point